main.py

```diff
@@ -0,0 +1,128 @@
+import cv2
+from matplotlib import image
+import mediapipe as mp
+import numpy as np
+
+mp_drawing = mp.solutions.drawing_utils
+mp_drawing_styles = mp.solutions.drawing_styles
+mp_holistic = mp.solutions.holistic
+mp_pose = mp.solutions.pose
+cap = cv2.VideoCapture('D:\\Tenis_04_07\\Wideonly\\zverev.mp4')
```

Comment:
All knows what cap in opencv means but generally don't use short forms. i would prefer 'capture' or even 'video_capture'